### ISSUE TITLE
Remove title attributes

### ISSUE THREAD

**Issue by stephenharris** [stephenharris/Event-Organiser#323]
Title attributes are useless. They are ignored by most screenreaders (by default), often just duplicate the link text and all they do is provide for a tooltip in browsers.

I used the following command to retrieve all title attributes:

     grep -onr --include=*.php --exclude-dir={dist,vendor,tests} 'title=' .

And it gave me:

- [ ] ./classes/class-eventorganiser-shortcodes.php:97:title=
- [ ] ./classes/class-eo-calendar-widget.php:275:title=
- [ ] ./classes/class-eo-calendar-widget.php:281:title=
- [ ] ./classes/class-eo-calendar-widget.php:283:title=
- [ ] ./classes/class-eo-calendar-widget.php:356:title=
- [x] ./event-organiser-venues.php:307:title=
- [x] ./event-organiser-add-ons.php:168:title=
- [x] ./event-organiser-add-ons.php:170:title=
- [x] ./event-organiser-add-ons.php:183:title=
- [ ] ./event-organiser-calendar.php:271:title=
- [ ] ./event-organiser-calendar.php:390:title=
- [x] ./includes/class-eo-admin-notice.php: 100:title=
- [x] ./includes/event-organiser-ajax.php:432:title=
- [x] ./includes/event-organiser-ajax.php:451:title=
- [x] ./includes/event-organiser-ajax.php:468:title=
- [x] ./includes/event-organiser-event-functions.php: 1234:title=
- [x] ./templates/shortcode-event-list.php:48:title=
- [x] ./templates/single-event.php:58:title=
- [x] ./templates/single-event.php:60:title=
- [x] ./templates/widget-event-list.php:44:title=

These should be looked at, and potentially removed, or where appropriate, moved to a `arial-label="..."`

**Comment by stephenharris** [comment]
Of the remaining cases:

- `event-organiser-calendar.php` is a false-positive. The title attribute is used for the title of the modal
- `class-eventorganiser-shortcodes.php` exposes an option to create a subscribe link with a title attribute (i.e. the title attribute generated is user-instigated)
- `class-eo-calendar-widget.php` is the only remaining case of title attributes used by the plug-in. It's rewriting would have front-end consequences (albeit minor) and would have to be considered in conjunction with #353. It's out of scope for this 'clean up' ticket.

Closing as fixed.
